docs/make.bat

```diff
@@ -0,0 +1,35 @@
+@ECHO OFF

```

Comment:
Was this a generated file? It looks like it's to support building the files on Windows? Maybe we don't need it?

Reply:
This was a generated file.
I'll dump it since we don't need it.